spec/activerecord-bitemporal/bitemporal_scope_spec.rb

```diff
@@ -777,6 +777,93 @@ class Article < ActiveRecord::Base
       end
     end
 
+    describe "ActiveRecord::Bitemporal.transaction_at" do
+      let(:transaction_datetime) { "2019/05/05".in_time_zone }
+      let(:relation) { ActiveRecord::Bitemporal.transaction_at(transaction_datetime) { User.all } }
+      it { is_expected.to have_valid_at(time_current, table: "users") }
+      it { is_expected.to have_transaction_at(transaction_datetime, table: "users") }
+
+      context "transaction_datetime is nil" do
+        let(:transaction_datetime) { nil }
+        it { is_expected.to have_bitemporal_at(time_current, table: "users") }
+      end
+
+      describe ".transaction_at" do
+        let(:user_transaction_datetime) { "2019/05/05".in_time_zone }
```

Comment:
If you want to test that the relation's transaction datetime takes precedence, I think `transaction_datetime` and `user_transaction_datetime` should be different.

Reply:
Thanks! Fix it.
https://github.com/kufu/activerecord-bitemporal/pull/82/commits/09e245f82ac55cbfdcecc4ce48a18049281e9f56